### PR TITLE
Incorporate Top Secret into core standards

### PR DESCRIPTION
**Pull request by mbryzek**

### CODE REVIEW

File: Lifecycle.md [hunk 1]
```diff
@@ -26,7 +26,27 @@ We pursue several practices to ensure our SDLC achieves its goals of providing v
...
+
+Examples:
+
+  - *How much documentation do we write?*
+  
+    Let's look at our favorite open source projects and see how their documentation looks. One example we like is the [react project](https://facebook.github.io/react/)
+
+  - *What level of testing do we use?*
+
+    Let's look at our favorite open source projects and see how their testing looks. One example we like is the [play framework itself](https://www.playframework.com/documentation)
```
Comment (chrhicks) [inline Lifecycle.md]: `play framework itself` -> `play framework`

File: Lifecycle.md [hunk 2]
```diff
@@ -48,19 +68,14 @@ To simplify local integration environments, and manage dependencies between appl
   - Unique port allocations, simplifying configuration for local integration environments
   - Declared dependency which informs which other applications, including databases, need to be running and healthy for a given application to work
 
-Docker artifacts must be produced as these artifacts are used in both the integration and production environments. Docker artifacts are built of the master branch of each repository. Docker artifacts are stored and managed by [Docker Hub](http://hub.docker.com).
+Docker artifacts must be produced as these artifacts are used in both the integration and production environments. Docker artifacts are built of the master branch off each repository. Docker artifacts are stored and managed by [Docker Hub](http://hub.docker.com).
```
Comment (chrhicks) [inline Lifecycle.md]: `Docker artifacts are built of the master branch off each repository.` -> `Docker artifacts are built from the master branch of each repository.`

File: Lifecycle.md (hunk 1 above)
Comment (mbryzek) [inline Lifecycle.md]: done

File: Lifecycle.md (hunk 2 above)
Comment (mbryzek) [inline Lifecycle.md]: done

**Comment by chrhicks** [comment]
lgtm

**Comment by mikeclimbrock** [comment]
lgtm

**Pull request merged**